vulyk/app.py

```diff
@@ -86,9 +85,10 @@ def types():
     Produces a list of available tasks types which are appropriate
     for current user.
     """
-    res = [t for t in TASKS_TYPES.keys() if g.user.is_eligible_for(t)]
+    tasks = [x.to_dict() for x in TASKS_TYPES.itervalues()
+             if g.user.is_eligible_for(x.type_name)]
```

Comment:
I think we should drop that method at all or turn it into helper (and call from view above)